api/app/signals/apps/signals/migrations/0115_auto_20200714_1023.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.13 on 2020-07-14 08:23
```

Comment:
I know this migration is also part of another PR. So we need to check if this is still needed when a PR get's merged. So in general the migrations can change.